client/cli/src/runner.rs

```diff
@@ -26,7 +26,14 @@ use log::info;
 use sc_service::{AbstractService, Configuration, Role, ServiceBuilderCommand, TaskType};
 use sp_runtime::traits::{Block as BlockT, Header as HeaderT};
 use sp_utils::metrics::{TOKIO_THREADS_ALIVE, TOKIO_THREADS_TOTAL};
-use std::{str::FromStr, fmt::Debug, marker::PhantomData, sync::Arc};
+use cli_utils::{AccountIdFor, AddressFor, IndexFor, BalanceFor, CallFor, RuntimeAdapter};
```

Comment:
err...

this means, sc-cli is now pulling in substrate-frame-cli-utils (see below), which means it pulls in frame-specifics. That is not okay. You must still be able to use the cli even if you are not building any frame-runtime. That might mean some cli-subkey-related features are not available, but that's alright – or even better, if these would be expressed as traits in primitives and could be implemented outside of frame ... maybe in a `sp-cli` set...